### ISSUE TITLE
unexpected token TK_COMMAND_BS as the argument of \ifile

### ISSUE THREAD

**Issue by aotto1968** [doxygen/doxygen#10937]
**Describe the bug**
```
test.doc:6: warning: unexpected token TK_COMMAND_BS as the argument of \ifile
```

**Screenshots**
![image](https://github.com/doxygen/doxygen/assets/27886664/5589f36e-ed92-4ff0-b257-9a65cabb657a)

**To Reproduce**
call: `doxygen Doxyfile.tag` and than `doxygen Doxyfile.bug-1`

**Expected behavior**
this depends… the core-problem is a @copydetails from a `tagfile` group target

**Version**
Doxygen version used: 1.10.0 (ebc57c6dd303a980bd19dd74b8b61c8f3f5180ca)
Linux linux02 5.14.21-150500.55.65-default #1 SMP PREEMPT_DYNAMIC Thu May 23 04:57:11 UTC 2024 (a46829d) x86_64 x86_64 x86_64 GNU/Linux

**Additional context**
Attachment → [doxygen_bug__unexpected_token_TK_COMMAND_BS.tar.gz](https://github.com/user-attachments/files/15815668/doxygen_bug__unexpected_token_TK_COMMAND_BS.tar.gz)

**Comment by albert-github** [comment]
Indeed the core problem is the `\copy...` from the tag file.
The tag files are intended to be used so one can "jump" to external (doxygen) generated documentation, so typically for commands like `\ref`. The tag files are not created for copying texts from external documentation to the current documentation (as this is unknown in the tag file). This can be nicely illustrated with:

**test.h**
```
/**
\defgroup MqContextC_FilterApi_C_API MqContextC_FilterApi_C_API
\{
\brief MqContextC - modify the data using the \e filter-technology ...
\}

**/


/// \brief the brief fie
void fie();
```

**test.doc**
```
/**

\page filter HOWTO Filter - The Basics

\section filter-intro                             INTRODUCTION
\copydoc MqContextC_FilterApi_C_API

\copydoc fie()

- \ref MqContextC_FilterApi_C_API
- \ref fie()

**/
```
which gives:
![image](https://github.com/doxygen/doxygen/assets/5223533/2d78a4ed-faca-4453-aec7-6be3d6452fab)


I used here `\copydoc` as we only have brief documentation and `\copydoc` handles brief and detailed documentation.
I used a slightly modified executable so the warning and the `"` in the output.

The fact that we don't get a warning for the `fie()` and the `\copybrief` of the group is that they have a "name" `<fie>` and  `<MqContextC_FilterApi_C_API>` whilst the ``copydetails` for the group remains empty (is is as if some detailed documentation is present but there isn't).

So in fact we have multiple problems:
- the warning
- the fact that the `\copy...` is silently ignored, though I think a warnings should be given.

**Comment by albert-github** [comment]
Regarding the warning problem I've just pushed a proposed patch, pull request #10939

**Comment by albert-github** [comment]
Regarding the fact that `\copy...` should not be used when reference is from a tag file: I've just pushed a proposed patch, pull request #10940

**Comment by albert-github** [comment]
Code of both pull requests has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
